thirdeye/thirdeye-frontend/app/mirage/fixtures/anomalyPerformance.js

```diff
@@ -1,13 +1,13 @@
 // Mock data that returns anomaly performance by application
 export default {
-  "totalAlerts":"7",   // displayed as "number of anomalies" on frontend
-  "totalResponses":"6",
-  "newTrend":"0",
-  "falseAlarm":"3",
-  "responseRate":"0.8571428571428571",
-  "trueAnomalies":"3",
-  "userReportAnomaly":"0",
-  "weightedPrecision":"0.46153846153846156",
-  "precision":"0.50",
-  "recall":"1.0"
+  'totalAlerts':'7',   // displayed as 'number of anomalies' on frontend
```

Comment:
nit: Yay, much cleaner than the double quotes`"`.  I understand that this is a JSON response, but since this currently lives in a JS file, let's omit the optional `quotes` around key property names.